HW_module_4_task3.py

```diff
@@ -0,0 +1,38 @@
+import re
+
+# Text variable
+message = """homEwork:
+	tHis iz your homeWork, copy these Text to variable.
+	You NEED TO normalize it fROM letter CASEs point oF View. also, create one MORE senTENCE witH LAST WoRDS of each existING SENtence and add it to the END OF this Paragraph.
+	it iZ misspeLLing here. fix“iZ” with correct “is”, but ONLY when it Iz a mistAKE.
+	last iz TO calculate nuMber OF Whitespace characteRS in this Text. caREFULL, not only Spaces, but ALL whitespaces. I got 87.
+"""
+
+# Last words of each existing sentence
+def find_all_last_word(a, b):
```

Comment:
In fact, here I will suggest using the default parameter: last_words_pattern='\s(\w+)?\.' 
If you name your function as find_all_last_word then by default it must find all last words in standard text.

Reply:
+ The same issue (parameters name)